nautobot_device_lifecycle_mgmt/utils.py

```diff
@@ -0,0 +1,35 @@
+"""Utility functions and classes used by the plugin."""
+
+
+from django.db.models import Count, Subquery, OuterRef
+from django.db.models.functions import Coalesce
+from django.urls import reverse
+from django.utils.safestring import mark_safe
+
+from nautobot.utilities.tables import LinkedCountColumn
+
+
+class M2MLinkedCountColumn(LinkedCountColumn):
```

Comment:
why not define it in `tables.py` if it's only used there?

Reply:
Good point. I'll move it to `tables.py` so it's closer to the code that uses it.